src/components/containers/PostListContainer.js

```diff
@@ -9,27 +9,26 @@ class PostListContainer extends Component {
     componentDidMount() {
         const { dispatch } = this.props;
 
-        dispatch(getPosts('http://localhost:9000/posts.json'));
+        dispatch(getPosts('/posts.json'));
```

Comment:
I think that make sense to put file name to constant and may be even move it to actions file ( to call just getPosts method in component instead of passing file name)